src/components/components/NavbarMobileLogoBar/NavbarMobileLogoBar.scss

```diff
@@ -0,0 +1,48 @@
+.NavbarMobileLogoBar {
+  display: none;
+}
+
+@media screen and (max-width: 480px) {
```

Comment:
I think we need to set a higher threshold for this, as the desktop menu still shows and is cut off for screen widths between 480 to about 540 px.

Reply:
I think Jasper said 640 or 680px today? But maybe we should make it a seperate ticket, so that someone can change it in every component that uses a mobile view